app/assets/javascripts/stradivari/tabs.js

```diff
@@ -29,4 +29,9 @@ jQuery(function() {
         loader.trigger('stradivari:tab:failed');
       });
   })
+
+  // activate tab if tab id is specified in the url#anchor
+  // no ifs required, and clicking the tab link works even with ajax tabs
+  $("[data-toggle='tab'][href='" + window.location.hash + "']")
```

Comment:
as discussed, please add an if to check whether the hash is present.

However I am not completely happy with this as we are abusing of the location hash.